content/vue/en/deploy.md

```diff
@@ -21,7 +21,7 @@ To deploy Storybook we first need to export it as a static web app. This functio
 ```javascript
 {
   "scripts": {
-    "storybook": "build-storybook -c .storybook -o .storybook-static"
+    "build-storybook": "build-storybook -c .storybook -s public -o storybook-static"
```

Comment:
The command should be `npm run storybook` or `yarn storybook` so this is fine only with `storybook`.
```suggestion
    "storybook": "build-storybook -c .storybook -s public -o storybook-static"
```

Reply:
so, if this change here it's necessary to change the script name in the following paragraph as well

> Now when you run Storybook via `npm run **build-**storybook`, it will output a static Storybook in the `storybook-static` directory.

Reply:
Good catch. Done.